config/core/webhooks/configmap-validation.yaml

```diff
@@ -32,6 +32,8 @@ webhooks:
   name: config.webhook.serving.knative.dev
   namespaceSelector:
     matchExpressions:
-    - key: serving.knative.dev/release
-      operator: Exists
+    - key: app.kubernetes.io/name
+      operator: In
+      values:
+      - knative-serving
```

Comment:
I realized this can be a `matchLabels` instead of a `matchExpressions`

https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements

Reply:
ditto for the other repos

Reply:
all done